_rules/auto-play-audio-has-control-mechanism-4c31df.md

```diff
@@ -0,0 +1,270 @@
+---
+id: 4c31df
+name: auto-play audio has a control mechanism
+rule_type: atomic
+description: |
+  auto-play audio or video must have a control mechanism.
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+This rule applies to any `audio` or `video` element that has:
+* an `autoplay` attribute that is equal to true, and
+* both `paused` and `muted` attributes equal to false, and 
+* either a `src` attribute or a child `source` element that references content with a duration of more than 3 seconds that contains audio.
+
+**Note**:
+
+The default value of both `paused` and `muted` attributes is `false`.
+
+## Expectation 1
+
+For each test target a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name), and is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
All objects in the accessibility will have an accessible name, though it might be the empty string. Should the rule expect there to be an accessible name that does not only consist of whitespace?